docs/content/preview/explore/linear-scalability/sharding-data.md

```diff
@@ -34,35 +34,43 @@ rightNav:
+The primary key for each row in the table uniquely identifies the location of the tablet in the row, as per the following following diagram:
 
 ![Sharding a table into tablets](/images/architecture/partitioning-table-into-tablets.png)
 
-By default, YugabyteDB creates eight tablets per node in the cluster for each table and automatically distributes the data across the various tablets, which in turn are distributed evenly across the nodes. In the [Try it out](#try-it-out) section, you'll explore how automatic sharding is done internally for tables. The system Redis table works in exactly the same way.
+By default, YugabyteDB creates eight tablets per node in the universe for each table and automatically distributes the data across various tablets, which in turn are distributed evenly across the nodes. In the [Examples](#Examples), you explore how automatic sharding is done internally for tables. The system Redis table works in exactly the same way.
 
 ## Sharding strategies
 
-Sharding is the process of breaking up large tables into smaller chunks called _shards_ that are spread across multiple servers. Essentially, a shard is a _horizontal data partition_ that contains a subset of the total data set, and hence is responsible for serving a portion of the overall workload. The idea is to distribute data that can't fit on a single node onto a cluster of database nodes.
+Sharding is the process of breaking up large tables into smaller chunks called shards spread across multiple servers. Essentially, a shard is a horizontal data partition that contains a subset of the total data set, and hence is responsible for serving a portion of the overall workload. The idea is to distribute data that cannot fit on a single node onto a cluster of database nodes.
 
-Sharding is also referred to as _horizontal partitioning_. The distinction between horizontal and vertical comes from the traditional tabular view of a database. A database can be split vertically&mdash;storing different table columns in a separate database, or horizontally&mdash;storing rows of the same table in multiple database nodes.
+Sharding is also referred to as horizontal partitioning. The distinction between horizontal and vertical comes from the traditional tabular view of a database. A database can be split vertically, storing different table columns in a separate database, or horizontally, storing rows of the same table in multiple database nodes.
 
-YugabyteDB currently supports two ways of sharding data: [hash](#hash-sharding) (also known as _consistent hash_) sharding, and [range](#range-sharding) sharding.
+YugabyteDB currently supports two ways of sharding data: [hash](#hash-sharding) (also known as consistent hash) sharding, and [range](#range-sharding) sharding.
+
+For additional information about sharding in YugabyteDB, see the following:  
+
+- [Architecture: sharding](../../../architecture/docdb-sharding/)
+
```

Comment:
extra space and switches from dashes to asterisks for bullets

Reply:
fixed